os/drivers/video/video_framebuff.c

```diff
@@ -0,0 +1,264 @@
+/****************************************************************************
+ * drivers/video/video_framebuff.c
+ *
+ *   Copyright 2018 Sony Semiconductor Solutions Corporation
```

Comment:
2019

Reply:
It is Sony license, we cannot change it. 
Same is used in nuttx as well.

Reply:
Add Samsung License header with year as 2019